### ISSUE TITLE
Jitsi doesn't start, message "NimbuzzAvatars"

### ISSUE THREAD

**Issue by Petrouchka** [jitsi/jitsi#163]
I installed JITSI jitsi-2.9.5477-x86.msi on fresh legal Windows XP Pro SP3 french version and jitsi-2.9.5477-x64.msi on Windows 7 Pro french version, in VirtualBox. After the installation, JITSI doesn't start, there's a message "NimbuzzAvatars" and nothing else.
![capture-1](https://cloud.githubusercontent.com/assets/3955499/10416190/e3651fbc-700b-11e5-8709-bc7436093c46.png)

Deleting the directory %userprofile%\.sip-communicator doesn't solve it. I restart with initial snapshot of Windows and had again the same problem.

Same problem with 2.9.5478 version but jitsi-2.8.5426 stable version works fine.

**Comment by ibauersachs** [comment]
Duplicate of #162.
Please don't open issue after issue. You already violated our rules the first time by not asking on the mailing list first. You can comment on the first issue.

**Comment by Petrouchka** [comment]
I opened a new issue because you closed my first issue as it was solved and without waiting my answer to your suggestion. Now, you closed my second issue without trying to answer to me and the issue isn't solved. Congratulations !
I will not waste my time anymore to test JITSI and making issues if you don't matter. And I will less use the mailing list before.
I will simply use another software that works, free or not.
